mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.tmpl

```diff
@@ -13278,3 +13355,63 @@ resource "google_container_cluster" "with_enterprise_config" {
 }
 `, projectID, clusterName, networkName, subnetworkName)
 }
+
+{{ if ne $.TargetVersionName `ga` -}}
+func testAccContainerCluster_withPodAutoscalingConfig(projectID, name, networkName, subnetworkName string, enable bool) string {
+
+  podAutoscalingBlock := ""
+  if enable { 
+    podAutoscalingBlock=`
+    pod_autoscaling {
+      hpa_profile = "NONE"
+    }
+  `
+  }
+
+  return fmt.Sprintf(`
+data "google_project" "project" {
+  provider   = google-beta
+  project_id = "%s"
+}
+resource "google_compute_network" "network" {
+  provider                 = google-beta
+  name                     = "%s"
+  auto_create_subnetworks  = false
+}
+resource "google_compute_subnetwork" "subnet" {
```

Comment:
Use a bootstrapped network & subnetwork here instead of creating new ones - search for BootstrapSharedTestNetwork and BootstrapSubnet in this file for examples.